### ISSUE TITLE
Proxy conf documentation

### ISSUE THREAD

**Issue by thomasdanan** [scality/metalk8s#267]
In case of using a proxy for the installation, we need to add the file all.yml in the folder inventory/<>/group_vars/
For no_proxy this should be set accordingly to the local IP setup

```
## Set these proxy values in order to update package manager and docker daemon to use proxies
http_proxy: "http://<ip>:<port>"
https_proxy: "http://<ip>:<port>"
## Refer to roles/kubespray-defaults/defaults/main.yml before modifying no_proxy
no_proxy: "127.0.0.1,10.0.0.0/8"
```

Also the file /etc/environment could be needed for Zenko. Not sure this is the right place for this paragraph

```
http_proxy=http://user;pass@<my-ip>:<my-port>
https_proxy=http://user;pass@<my-ip>:<my-port>
no_proxy=127.0.0.1,10.0.0.0/8
```

**Comment by thomasdanan** [comment]
Well for no_proxy we should also add localhost

**Comment by NicolasT** [comment]
This has been merged (mostly) in #328, however the documentation may not be entirely correct depending on the proxy configuration/location.

This certainly relates to #123
